answers/w11.md

```diff
@@ -0,0 +1,183 @@
+### What are constructor functions? How do we use them?
+
+Constructor functions define a new instance of a Javascript object. We create a Javascript object using the `new` keyword.  
+
+```js
+var Planet = function( info ){
+  this.name = info.name;
+  this.distance = info.distance;
+}
+
+var neptune = new Planet({ name: 'Neptune', distance: 'far away' });
+```
+
+### What is a prototype? Why do we care about them?
+
+Prototypes define the default -- or "blueprint" -- values and methods of an object.  
+
+### What action do you expect this route to perform: `app.post('/authors', function() {...}`?
+
+It is a `POST` request, so it will create a new `author` in our database.
+
+### Write the url to request all authors from a Rails app, formatted as JSON.
+
+`http://localhost:3000/authors.json`
+
+### Create a media query that supports most mobile phones.
+
+```css
+@media only screen and (max-device-width: 480px) {
```

Comment:
Should we orientation?
```
/* #### Mobile Phones Portrait #### */
@media screen and (max-device-width: 480px) and (orientation: portrait){
  /* some CSS here */
}
```

References: 
- http://www.javascriptkit.com/dhtmltutors/cssmediaqueries2.shtml
- http://stackoverflow.com/questions/8549529/what-is-the-difference-between-screen-and-only-screen-in-media-queries